Automattic/wp-calypso
# HelpCenter: Add HelpCenter to the new WPSupport page

## Proposed Changes

Adds Help Center to the new WPCOM support page. More [context here](https://github.com/Automattic/wpsupport3/issues/232)

## Testing Instructions
 
- Pull and run this branch
- Navigate to `apps/editing-toolkit` and run `yarn dev --sync` to sync your sandbox
- Make sure to sandbox wordpress.com/support
- Navigate to `https://wordpress.com/support`
- Check if the Help Center is rendered on the admin bar
- Help Center should work fully and normally 

<img width="2045" alt="image" src="https://user-images.githubusercontent.com/2653810/221893563-118894b7-53f2-4f30-8c6f-4af453c1fdaf.png">
